extensions/amp-audioburst-audio/0.1/test/validator-amp-audioburst-audio.html

```diff
@@ -0,0 +1,43 @@
+<!--
+  Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS-IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the license.
+-->
+<!--
+  Test Description:
+  This tests validation for the amp-audioburst-audio tag.
+-->
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>amp-audioburst-audio example</title>
+  <link rel="canonical" href="amps.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    amp-audioburst-audio {
+      font-size: 14px;
+      font-family: sans-serif;
+    }
+  </style>
+  <script async custom-element="amp-audioburst-audio" src="https://cdn.ampproject.org/v0/amp-audioburst-audio-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+  <amp-audioburst-audio autoplay layout="fixed-height" height="315"
+    src="https://sapi.audioburst.com/audio/repo/play/web/R6rDYxkvqVOK.mp3?userId=unknown&appName=search.audioburst&qid=0"
+    fullShow="http://storageaudiobursts.blob.core.windows.net/temp/11444_2018081406_t.mp3" fullShowPosition="1856.34"
+  >
+  </amp-audioburst-audio>
+</body>
```

Comment:
Consider adding an example of a file that should fail validation, like a player with `fullShow` set but is missing `fullShowPosition`.

Reply:
done